UnityProject/Assets/Scripts/Managers/SubSceneManager/SubSceneManager.SceneList.cs

```diff
@@ -217,37 +219,47 @@ IEnumerator ServerLoadAwaySite(SubsceneLoadTimer loadTimer)
 	public IEnumerator LoadSyndicate()
 	{
 		if (SyndicateLoaded) yield break;
+		var pickedMap = additionalSceneList.defaultSyndicateScenes.PickRandom();;
```

Comment:
codacy is crying because the double ``;``

Reply:
how did that end up there